CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ newer version, we recommend that you to go through the list of changes.
 
 ### Improvements and fixes
 
+* Added complete support for CKD simulations in AFGL 1986 reference atmospheres ({ghpr}`185`).
```

Comment:
I suggest to rephrase this like as follows (a little more descriptive regarding your PR contents):
```suggestion
* Add support for all missing AFGL 1986 reference atmospheres in CKD mode ({ghpr}`185`).
```